CMakeLists.txt

```diff
@@ -328,6 +328,11 @@ set (
     "adu"
     CACHE STRING "The effective group name for adu-shell process.")
 
+set (
+    EIS_PIPE_PATH
+    "/tmp/eis_pipe"
```

Comment:
Since this will contains device connection info, let's use the path that accessible by snap_aziot_du group.
Perhaps /var/lib/adu/<eis_response_file_name> ?

Reply:
"/var/lib/adu/eis_response_pipe" works?